package/gluon-web-network/luasrc/lib/gluon/web/model/admin/network.lua

```diff
@@ -122,15 +122,17 @@ if sysconfig.lan_ifname then
 	end
 end
 
-if uci:get('system', 'gpio_switch_poe_passthrough') then
-	local s = f:section(Section)
-	local poe_passthrough = s:option(Flag, "poe_passthrough", translate("Enable PoE passthrough"))
-	poe_passthrough.default = uci:get_bool("system", "gpio_switch_poe_passthrough", "value")
-
-	function poe_passthrough:write(data)
-		uci:set('system', 'gpio_switch_poe_passthrough', 'value', data)
+uci:foreach("system", "gpio_switch", function(s)
+	if s[".name"]:match("poe") then
+		local section = f:section(Section)
```

Comment:
I don't think it makes much sense to put each option into a separate Section.

Note that it will be necessary to give each option a unique identifier instead of "poe_passthrough" (for example `s[".name"]`) when they're added to the same Section.